### PR TITLE
CSS File

### PR DESCRIPTION
**Pull request by rogeliomonterde**
hi this is my first project in microverse, I am doing my best to achieve the goal: 
set up linters
Add HTML file with tags
Add CSS file with styling
Add test.md file
Ignored some files in gitignore

### CODE REVIEW

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @rogeliomonterde, 
I'm Andres.


While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of GitHub flow. 💯 

## Required Changes ♻️
_Check the comments under the review._

You did a good job following the requirements 🚀. However, there are still some things that you need to improve in order to move on:

- [ ] I see that you tried to set up the linters, however, they are not working at the moment due to a typo in the `.github/workflows` folder. You missed the `"s"`. Please consider following [this spet-by-step tutorial](https://github.com/microverseinc/linters-config/tree/master/html-css) paying extra attention to naming the files and folders correctly in order to have linters up and running! 🤓 🛠️ 

- [ ] Please consider using the [Microverse's Readme Template](https://github.com/microverseinc/readme-template). During the whole program you're going to be asked to write documentation for your projects, so please have a look at it, get familiar with it, and consider adding it to your project (copying its contents but using your project information). 🤓 🛠️ 

- [ ] Also, please consider adding a better description for your pull request describing in detail the changes you are implementing. 🤓 🛠️ 

image

At the moment `"hi"` is not a good description, please consider editing it! 🚀 

- [ ] According to the requirements the `test.md` file should be ignored by git but your `.gitignore` file is empty, please do it! 🤓 🛠️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rogeliomonterde** [comment]
Hi I could solve the problems on my local repository but I have problems updating the file here

**Comment by rogeliomonterde** [comment]
I dont know if I did it correctly
when i try to push the main branch comes this errror:

o github.com:rogeliomonterde/hello.git
 ! [rejected]        main -> main (fetch first)
error: failed to push some refs to 'github.com:rogeliomonterde/hello.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

File: README.md [hunk 1]
```diff
@@ -1,267 +1,532 @@
...
+- License
+
+
+After you're finished please remove all the comments and instructions!
+-->
+
+  
+
+  <h3><b>Hello README </b></h3>
+
```
Comment (DuaneDave) [inline README.md:295]: Good job implementing the prescribed readme template but currently you have it duplicated. Please consider removing from line 271 downwards to make the documentation not repetitive and cumbersome.

File: README.md [hunk 2]
```diff
@@ -1,267 +1,532 @@
...
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
```
Comment (DuaneDave) [inline README.md:88]: Consider updating the tech stack provided in the template to the ones you have actually used (`html` and `CSS`) in order to have a readme that is consistent with the project

File: README.md [hunk 3]
```diff
@@ -1,267 +1,532 @@
...
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[beautiful colors]**
+- **[beautiful background]**
+- **[beautiful name]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (DuaneDave) [inline README.md:101]: Please add at least three cool features about this project here as they will easily draw the attention of any user to what they should expect from this project.

File: README.md [hunk 4]
```diff
@@ -1,267 +1,532 @@
...
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
```
Comment (DuaneDave) [inline README.md:135]: This section should include a step-by-step guide as to what a user should do in order to get this project working  on their local machine. Kindly add such so new users of this project will know how to get it all set up 😄

File: README.md [hunk 5]
```diff
@@ -1,267 +1,532 @@
...
+### Run tests
+
+To run tests, run the following command:
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
```
Comment (DuaneDave) [inline README.md:179]: Also, I recommend that you hint to users on how tests could be run on this project. Kindly add the commands needed to check for linter errors here in order to help new users get started.

File: README.md [hunk 6]
```diff
@@ -1,267 +1,532 @@
...
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (DuaneDave) [inline README.md:214]: Sometimes, users would love to follow or contact you on your social handles, in view of this, I suggest including your social handles here.

File: README.md [hunk 7]
```diff
@@ -1,267 +1,532 @@
...
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
+
```
Comment (DuaneDave) [inline README.md:225]: Please add at least three future features for this project here in order to keep users noticed of future changes to come.

File: README.md [hunk 8]
```diff
@@ -1,267 +1,532 @@
...
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (DuaneDave) [inline README.md:257]: Please include an acknowledgement giving credit to those whose ideas or code were used in this project to avoid plagiarizing

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @rogeliomonterde 👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ Linters are set up
✔️ All files are present


 ## Required Changes ♻️
- To keep our codebase free of mistakes, it is advisable that all linters checks be green. kindly resolve the stylint error to follow this pattern of mistake-free code.

see below

61BE4693-88F5-4980-A71E-CBCFB0B9E0EA



- As previously requested, do consider adding a better description for your pull request describing in detail the changes you are implementing in order to make it clear the activities done for this project.
Instead of hi, enumerate the things done in this milestone eg
    - set up linters
    - Add HTML file with tags
    - Add CSS file with styling
    - Add test.md file
    - Ignored some files in gitignore

Click on the three dots by the right corner to edit

55BAB2D1-92BA-4999-9B5D-479463D2F712


_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DuaneDave** [comment]
> I dont know if I did it correctly when i try to push the main branch comes this errror:
> 
> o github.com:rogeliomonterde/hello.git ! [rejected] main -> main (fetch first) error: failed to push some refs to 'github.com:rogeliomonterde/hello.git' hint: Updates were rejected because the remote contains work that you do hint: not have locally. This is usually caused by another repository pushing hint: to the same ref. You may want to first integrate the remote changes hint: (e.g., 'git pull ...') before pushing again. hint: See the 'Note about fast-forwards' in 'git push --help' for details.

Whenever you encounter this issue again, please run git pull as the beach you are pushing have some changes that you do not have locally then run git push

File: README.md [hunk 9]
```diff
@@ -1,267 +1,532 @@
...
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
```
Comment (bobb-Rob) [inline README.md:87]: As mentioned by the previous reviewer, please consider updating the tech stack section according to your project. You will do this for all your microverse projects. 
image

File: README.md [hunk 10]
```diff
@@ -1,267 +1,532 @@
...
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```
Comment (bobb-Rob) [inline README.md:212]: Please edit this contact section as well and add your social media handles.

File: index.html [hunk 11]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
```
Comment (bobb-Rob) [inline index.html:4]: - [ ] As mentioned by the previous reviewer, your Pull Request title and summary/description is still not written professionally, Kindly follow the instructions on [the previous review](https://github.com/rogeliomonterde/hello/pull/1#pullrequestreview-1243113454).
- [ ] There are conflicts in your code files. [Please read through this article to learn to resolve code conflicts](https://akshayranganath.github.io/Git-Pull-Handling-Merge-Conflict/).
- [ ] Kindly go through all the changes requested by other reviewers and make changes accordingly. Thank you

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️

Hi @rogeliomonterde , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**